### DISCUSSION TITLE
Noob question...

### DISCUSSION THREAD

**Discussion by jferris-** [OlivierGaland/CashFactory#22] [Q&A] [answered]
Sorry to be that person...

The step 'Update .env file with the account details (put your account informations in defined environment variables)', where is this .env file located that I need to edit, please?

I just edit the .yaml file, right?

**Comment by OlivierGaland** [comment, marked as answer]
Hello,

To install the stack correctly you must update the .env file to put the apps credentials you had once you registered, you may need to update docker-compose.yml only in specific cases as disabling one app you don't want to run, or if you need some tuning (like using other ports for GUI, in that case basic knowledge of docker and docker-compose is welcome)

The file is located in same dir as the docker-compose.yml and included in the release, you just need to create your account on the cash apps then put your credentials in it. (as it contains your credentials, do not share or show it to anyone)

Without filling it the apps won't start correctly as they won't know which account to use (except for earnapp that is set up in another way -see readme-)

![image](https://user-images.githubusercontent.com/26048157/163497656-4b2af516-ac47-4bd4-9a62-874e3114628d.png)

If you cannot see it, maybe your user interface is not showing it. Type ls -al .env in a shell to confirm it is here. If you are not reluctant to use vi, just type vi .env and you should be able to edit it, if you use a text editor and you cannot see it when your search for this file, go in the correct directory and write the file name ".env" in the window and click open, it should find it.
